lib/oneview-sdk/resource/api200/rack.rb

```diff
@@ -51,6 +51,13 @@ def delete(*)
         unavailable_method
       end
 
+      # Updates a rack
+      # @param [Hash] attributes attributes to be updated
+      # @return [OneviewSDK::Rack] self
+      def update(attributes = {}, header = self.class::DEFAULT_REQUEST_HEADER)
```

Comment:
Question: What exactly are does `self.class::DEFAULT_REQUEST_HEADER` resolves to? `{}.freeze`?
Can't we just `header  = { 'If-Match' => '*' }` here?

Reply:
Really! Good catch.